examples/Migration/CampaignManagement/CreateCompleteCampaignAdwordsApiOnly.php

```diff
@@ -0,0 +1,380 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Ads\GoogleAds\Examples\Migration\CampaignManagement;
+
+use Google\AdsApi\AdWords\AdWordsServices;
+use Google\AdsApi\AdWords\AdWordsSession;
+use Google\AdsApi\AdWords\v201809\cm\AdGroup;
+use Google\AdsApi\AdWords\v201809\cm\AdGroupAd;
+use Google\AdsApi\AdWords\v201809\cm\AdGroupAdOperation;
+use Google\AdsApi\AdWords\v201809\cm\AdGroupAdRotationMode;
+use Google\AdsApi\AdWords\v201809\cm\AdGroupAdService;
+use Google\AdsApi\AdWords\v201809\cm\AdGroupAdStatus;
+use Google\AdsApi\AdWords\v201809\cm\AdGroupCriterion;
+use Google\AdsApi\AdWords\v201809\cm\AdGroupCriterionOperation;
+use Google\AdsApi\AdWords\v201809\cm\AdGroupCriterionService;
+use Google\AdsApi\AdWords\v201809\cm\AdGroupOperation;
+use Google\AdsApi\AdWords\v201809\cm\AdGroupService;
+use Google\AdsApi\AdWords\v201809\cm\AdGroupStatus;
+use Google\AdsApi\AdWords\v201809\cm\AdRotationMode;
+use Google\AdsApi\AdWords\v201809\cm\AdvertisingChannelType;
+use Google\AdsApi\AdWords\v201809\cm\BiddableAdGroupCriterion;
+use Google\AdsApi\AdWords\v201809\cm\BiddingStrategyConfiguration;
+use Google\AdsApi\AdWords\v201809\cm\BiddingStrategyType;
+use Google\AdsApi\AdWords\v201809\cm\Budget;
+use Google\AdsApi\AdWords\v201809\cm\BudgetBudgetDeliveryMethod;
+use Google\AdsApi\AdWords\v201809\cm\BudgetOperation;
+use Google\AdsApi\AdWords\v201809\cm\BudgetService;
+use Google\AdsApi\AdWords\v201809\cm\Campaign;
+use Google\AdsApi\AdWords\v201809\cm\CampaignOperation;
+use Google\AdsApi\AdWords\v201809\cm\CampaignService;
+use Google\AdsApi\AdWords\v201809\cm\CampaignStatus;
+use Google\AdsApi\AdWords\v201809\cm\CpcBid;
+use Google\AdsApi\AdWords\v201809\cm\ExpandedTextAd;
+use Google\AdsApi\AdWords\v201809\cm\Keyword;
+use Google\AdsApi\AdWords\v201809\cm\KeywordMatchType;
+use Google\AdsApi\AdWords\v201809\cm\Money;
+use Google\AdsApi\AdWords\v201809\cm\NetworkSetting;
+use Google\AdsApi\AdWords\v201809\cm\Operator;
+use Google\AdsApi\AdWords\v201809\cm\UrlList;
+use Google\AdsApi\AdWords\v201809\cm\UserStatus;
+
+require __DIR__ . '/../vendor/autoload.php';
+
+/**
+ * This code example is the first in a series of code examples that shows how to create
+ * a Search campaign using the AdWords API, and then migrate it to Google Ads API one
+ * functionality at a time. See other examples in this directory for code examples in various
+ * stages of migration.
+ *
+ * This code example represents the initial state, where the AdWords API is used to create a
+ * campaign budget, a Search campaign, ad groups, keywords and expanded text ads. None of the
+ * functionality has yet been migrated to the Google Ads API.
+ */
+class CreateCompleteCampaignAdwordsApiOnly
```

Comment:
Technically, the file name should be "AdWordsApi" not "AdwordsApi".

Reply:
Done.